### DISCUSSION TITLE
Question about dimension Mul and Div parameters

### DISCUSSION THREAD

**Discussion by yasharne** [netdata/netdata#13256] [Q&A]
Hi
I see this piece of code: https://github.com/netdata/go.d.plugin/blob/master/modules/mysql/charts.go#L810
I wonder why there is a multiply of `100` then a division of `1000 * m.UpdateEvery`?

**Comment by ilyam8** [comment]
Hi, @yasharne. `Cpu_time` is 

> [The cumulative CPU time elapsed while servicing this user's connections](https://mariadb.com/kb/en/information-schema-user_statistics-table/).

- divisor `1000 * m.UpdateEvery` because we multiply the value by [1000 during data collection](https://github.com/netdata/go.d.plugin/blob/7e29023ba805e6bfc8a3e6c972851effbb4ff203/modules/mysql/collect_user_statistics.go#L70-L73).
- multiplier `100` to convert the result to a percentage.
